upgrading/automated_upgrades.adoc

```diff
@@ -525,6 +530,17 @@ link:http://docs.ansible.com/ansible/playbooks_delegation.html#delegation[`deleg
 If your {product-title} cluster uses a mixed environment or gcePD storage,
 you need to take more steps before you upgrade it.
 
+[[special-considerations-for-large-scale-upgrades]]
+==== Special considerations for large-scale upgrades
+
+For large-scale cluster upgrades, which involve at least 10 worker nodes and
+thousands of projects and pods, pre- and post- storage migration must be
+completed before and after the upgrade playbooks are run. Otherwise, the upgrade
+process will fail.
+
+See link:https://access.redhat.com/articles/3208381[Recommendations for
+large-scale OpenShift upgrades] for further guidance.
```

Comment:
Change "See" to be more specific:

"Refer to the 'Running the pre and post API server model object migration outside of the upgrade window' section of the link:https://access.redhat.com/articles/3208381[Recommendations for large-scale OpenShift upgrades] for further guidance."